core/src/main/java/com/linecorp/armeria/common/stream/StreamMessage.java

```diff
@@ -143,6 +148,52 @@
         }
     }
 
+    /**
+     * Creates a new {@link StreamMessage} that streams the specified {@link File}.
+     * The default buffer size({@value PathStreamMessage#DEFAULT_FILE_BUFFER_SIZE}) is used to read
```

Comment:
Could you add a description of the relationship between the buffer size and demand?